tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js

```diff
@@ -88,6 +88,12 @@ context('OpenCV. Intelligent cissors. Histogram Equalization.', () => {
             cy.get('.cvat_canvas_interact_intermediate_shape').should('have.attr', 'fill-opacity', 0);
             cy.get('.cvat-appearance-selected-opacity-slider').click('right');
             cy.get('.cvat_canvas_interact_intermediate_shape').should('have.attr', 'fill-opacity', 1);
+            // Intelligent scissors blocking feature
+            cy.get('.cvat_canvas_threshold').should('exist');
+            cy.contains('span', 'Block').click();
+            cy.get('.cvat_canvas_threshold').should('not.exist');
+            cy.contains('span', 'Block').click();
+            cy.get('.cvat_canvas_threshold').should('exist');
```

Comment:
Please, check that the block actually work (it creates exactly line user draws without adding any new points)

Reply:
Reworked. I allocated it to a separate test. Added checks.